unidist/run.py

```diff
@@ -0,0 +1,247 @@
+            "-n",
+            "1",
+            "-wdir",
+            workers_dir,
+            "python3",
+            unidist_home_path + "/unidist/core/backends/mpi/core/monitor.py",
+        ]
+
+        def get_worker_command(num_workers):
+            return [
+                "-n",
+                num_workers,
+                "-wdir",
+                workers_dir,
+                "python3",
+                unidist_home_path + "/unidist/core/backends/mpi/core/worker.py",
+            ]
+
+        command += [hosts_str] + command_executor + [":"] + command_monitor
+
+        for n in num_workers:
+            command += [":"] + get_worker_command(n)
+    else:
+        command = [executor, script]
+
+    return command
+
+
+def main():
+    """Run unidist application."""
+    usage_examples = [
+        "\n\tIn case 'unidist' is installed, use binary:",
+        "\n\tunidist script.py  # Ray backend is used",
+        "\n\tunidist script.py --backend MPI  # MPI backend is used",
+        "\n\tunidist script.py --executor pytest -b Dask  # Dask backend is used, running using 'pytest'",
+        "\n\n\tTo run from sources use 'unidist/run.py':",
+        "\n\tpython3 unidist/run.py script.py -b MPI --num_workers=16 --hosts localhost  # MPI backend uses 16 workers on 'localhost' node",
+        "\n\tunidist script.py -b MPI --num_workers=2 4 --hosts localhost x.x.x.x  # MPI backend uses 2 workers on 'localhost' and 4 on 'x.x.x.x'",
+    ]
+    parser = argparse.ArgumentParser(
+        description="Run python code with 'unidist' under the hood.",
+        usage="".join(usage_examples),
+    )
+
+    required_args_group = parser.add_argument_group("required arguments")
+    mpi_args_group = parser.add_argument_group("MPI-backend specific arguments")
+    required_args_group.add_argument("script", help="set script to be run")
+    parser.add_argument(
+        "-b",
+        "--backend",
+        type=str,
+        choices=["Ray", "MPI", "Dask", "MultiProcessing", "Python"],
+        default="Ray",
+        help="set an execution backend. Default is 'Ray'",
+    )
+    parser.add_argument(
+        "-e",
+        "--executor",
+        type=str,
+        default="python3",
+        help="set an executable to run. Default is 'python3'",
+    )
+    mpi_args_group.add_argument(
+        "-num_workers",
+        "--num_workers",
+        default=["default"],
+        nargs="+",
+        help="set a number of workers per node in a cluster. Can accept multiple values in a case of working on the cluster. Default is equal to the number of CPUs on a head node.",
+    )
+    mpi_args_group.add_argument(
+        "-hosts",
+        "--hosts",
+        default=["localhost"],
+        nargs="+",
+        help="set a node ip address to use. Can accept multiple values in a case of working on the cluster. Default is 'localhost'.",
+    )
+    args = parser.parse_args()
+    kwargs = vars(args)
+
+    os.environ["UNIDIST_BACKEND"] = kwargs["backend"]
```

Comment:
We will use `Backend.put(kwargs["backend"])` here after #21 is merged.

Reply:
But in this case we should add `import unidist ...` in this file. Are you sure that a correct backend will be propagated in a subprocess?

Reply:
That should work but needs to be verified on different OSs.

Reply:
Using of setting up with `unidist.config...` here isn't possible because main process and subprocess have different unique ids of entities (`CpuCount`, `Backends` etc) in `unidist`. Setting up via env vars will be used.